ghost/i18n/locales/ca/comments.json

```diff
@@ -3,69 +3,69 @@
     "Discussion": "Debat",
     "Edit": "Editar",
     "Edit this comment": "Editar aquest comentari",
     "edited": "editat",
     "Enter your name": "Escriu el teu nom",
-    "Expertise": "Conéixement",
+    "Expertise": "Expertesa",
     "Founder @ Acme Inc": "Fundador @ Acme Inc",
     "Full-time parent": "Pare a temps complert",
     "Head of Marketing at Acme, Inc": "Cap de màrqueting a Acme, Inc",
-    "Hidden for members": "",
-    "Hide": "Amaga",
-    "Hide comment": "Amaga el comentari",
+    "Hidden for members": "Ocult pels membres",
+    "Hide": "Oculta",
+    "Hide comment": "Oculta el comentari",
     "Jamie Larson": "Jamie Larson",
     "Join the discussion": "Uneix-te al debat",
     "Just now": "Just ara",
-    "Local resident": "Resiudent local",
+    "Local resident": "Resident local",
     "Member discussion": "Debat de membres",
     "Name": "Nom",
     "Neurosurgeon": "Neurocirurgià",
-    "Once deleted, this comment can’t be recovered.": "",
+    "Once deleted, this comment can’t be recovered.": "Un cop esborrat, aquest comentari no es pot recuperar",
     "One hour ago": "Fa una hora",
     "One min ago": "Fa un minut",
-    "removed": "",
-    "Replied to": "",
+    "removed": "Esborrat",
+    "Replied to": "Respost a",
     "Reply": "Respondre",
-    "Reply to": "",
+    "Reply to": "Respondre a",
     "Reply to comment": "Respondre al comentari",
     "Report": "Reportar",
     "Report comment": "Reportar comentari",
     "Report this comment?": "Reportar aquest comentari?",
-    "Save": "Desa",
+    "Save": "Desar",
     "Sending": "Enviant",
-    "Sent": "Enviar",
+    "Sent": "Enviat",
     "Show": "Mostra",
-    "Show {{amount}} more replies": "Mostra {{amount}} respostes més",
-    "Show {{amount}} previous comments": "Mostra {{amount}} comentaris previs",
-    "Show 1 more reply": "Mostra 1 resposta més",
-    "Show 1 previous comment": "Mostra 1 comentari previ",
-    "Show comment": "Mostra el comentari",
-    "Sign in": "Inici de sessió",
+    "Show {{amount}} more replies": "Mostrar {{amount}} respostes més",
+    "Show {{amount}} previous comments": "Mostrar {{amount}} comentaris previs",
+    "Show 1 more reply": "Mostrar 1 resposta més",
+    "Show 1 previous comment": "Mostrar 1 comentari previ",
+    "Show comment": "Mostrar comentari",
+    "Sign in": "Iniciar sessió",
     "Sign up now": "Registrar-se ara",
-    "Sort by": "",
-    "Start the conversation": "Comença la conversa",
-    "This comment has been hidden.": "Aquest comentari s'ha amagat.",
+    "Sort by": "Ordenar per",
+    "Start the conversation": "Començar la conversa",
+    "This comment has been hidden.": "Aquest comentari s'ha ocultat.",
     "This comment has been removed.": "Aquest comantari s'ha esborrat.",
-    "Upgrade now": "Millora ara",
+    "Upgrade now": "Actualitzar ara",
     "Yesterday": "Ahir",
     "Your request will be sent to the owner of this site.": "La vostra sol·licitud s'enviarà al propietari d'aquest lloc."
```

Comment:
Vostra is inconsistent with the informal language used elsewhere, no?

Reply:
You're right. I somehow missed this one. Should be changed to "teva". Thanks for spotting it!